Document/0x05e-Testing-Cryptography.md

```diff
@@ -8,21 +8,76 @@ A general rule in app development is that one should never attempt to invent the
 
 Android developers don't need to bother much with the intricate details of cryptography most of the time. However, even when using standard algorithms can be affected if misconfigured. 
```

Comment:
"However, even when using standard algorithms can be affected if misconfigured."
Maybe change to this "However, even standard algorithms can be affected if misconfigured. "